vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericSetVariableSteps.java

```diff
@@ -149,7 +149,7 @@ public void saveContextElementAttributeValueToVariable(String attributeName, Set
      * </ul>
      * @param variableName A name under which the value should be saved
      */
-    @When("I save `$attributeName` attribute value of element located `$locator` to $scopes variable `$variableName`")
+    @When("I save `$attributeName` attribute value of element located by `$locator` to $scopes variable `$variableName`")
     public void saveAttributeValueOfElementByLocatorToVariable(String attributeName, Locator locator,
```

Comment:
I would like to replace `assertNotNull` used by this step with more user-friendly assertion